src/autopas/selectors/tuningStrategy/TuningStrategyInterface.h

```diff
@@ -36,9 +36,10 @@ class TuningStrategyInterface {
    * The new configuration can be obtained by getCurrentConfiguration. It is the configuration which is either the next
    * configuration to test (=true) or the optimum (=false).
    *
+   * @param currentInvalid set to true to mark current configuration as invalid
```

Comment:
```suggestion
   * @param currentInvalid Tells the tune() function that the currently selected configuration is invalid. This can be used to avoid getting stuck in an invalid optimum.
```